instructure/canvas-ios
# Add new help menu UI

refs: MBL-14959
affects: Parent, Student, Teacher
release note: Add new help menu UI.

test plan:
- Open help menu.
- Observe new design.
- Note help item subtitles are displayed.
- All menu items should work just as before.
<table>
<tr>
<th>Old</th>
<th>New</th>
</tr>
<tr>
<td><img src="https://user-images.githubusercontent.com/72396990/99515032-b8f29d00-298c-11eb-9539-c9067096e72a.png"></td>
<td><img src="https://user-images.githubusercontent.com/72396990/99514064-9d3ac700-298b-11eb-84fa-a5e3e9b56af0.png"></td>
</tr>
</table>
